xml/System.IO/BinaryReader.xml

```diff
@@ -1114,8 +1114,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Reads an integer 7 bits at a time.</summary>
```

Comment:
The API is called `Read7BitEncodedInt64`. I'm wondering if the word `integer` is still right here?

Reply:
```suggestion
        <summary>Reads a number 7 bits at a time.</summary>
```